test/models/work_test.rb

```diff
@@ -0,0 +1,52 @@
+require "test_helper"
+
+describe Work do
+  describe 'validations' do
+    before do
+      @work = Work.new(
+        title: "A Great Book"
+      )
+    end
+
+    it "passes validations with good data" do
+      expect(@work).must_be :valid?
+    end
+
+    it "rejects work with repeated title" do
+      duplicate_title = Work.first.title
+      @work.title = duplicate_title
+
+      result = @work.valid?
+
+      expect(result).must_equal false
+      expect(@work.errors.messages).must_include :title
+    end
+  end
+
+  describe 'custom methods' do
+    it 'sorts media by number of votes' do
+      Vote.all = {
+        one: {
```

Comment:
There's a lot of interesting test cases for your custom `Work` methods missing here, particularly for the `top_ten` method. For example, I would ask:

- What if there are no works of that category?
- What if there are less than 10 works?
- What if there's a tie for last place, e.g. works 9, 10 and 11 all have 0 votes?